.github/workflows/checkstyle.yaml

```diff
@@ -17,6 +17,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.17.2"
```

Comment:
do you want to leave a comment in WORKSPACE that points to this file and mentions these versions should be kept in sync?

Reply:
Done